variables.tf

```diff
@@ -57,6 +57,12 @@ variable "agents_min_count" {
   description = "Minimum number of nodes in a pool"
 }
 
+variable "max_surge" {
+  type        = number
```

Comment:
Since we can set a percentage value to this variable, should we use `string` as type?

---

Thanks to your pr, I found that the [current variable's type for additional node pool](https://github.com/Azure/terraform-azurerm-aks/blob/main/variables.tf#L780-L782) is also incorrect, would you please correct it in this pr for us? Thanks!

Reply:
Yes, I will update it as well. Having string makes perfect sense so we can use both number or percentage.